keyboards/keychron/q2_pro/config.h

```diff
@@ -47,10 +47,10 @@
 #        define LED_DRIVER_SHUTDOWN_PIN C14
 
 #        define HOST_LED_MATRIX_LIST \
-            { 16, 17, 18 }
+            { 15, 16, 17 }
 
 #        define BAT_LEVEL_LED_LIST \
-            { 16, 17, 18, 19, 20, 21, 22, 23, 24, 25 }
+            { 15, 16, 17, 18, 19, 20, 21, 22, 23, 24 }
```

Comment:
```suggestion
            { 1, 2, 3, 4, 5, 6, 7, 8, 9, 10}
```
I think we should use key 1 to 0 to indicate the battery level.

Reply:
I agree, though I created the initial commit to match the behavior of the stock firmware from here: https://www.keychron.com/blogs/archived/how-to-factory-reset-or-flash-your-qmk-via-enabled-keychron-q2-pro-keyboard